build.gradle

```diff
@@ -84,6 +84,8 @@ sourceSets {
     main {
         java {
             srcDir "common"
+            srcDir "common_old_license"
```

Comment:
do we want to re-add the old licence code as sources or is this some kind of leftover?

Reply:
Some of the filler pattern code isn't yet MPLv2, so this folder is for new (currently used) source files that are under MMPLv1

Reply:
we probably want to get those under MMPLv2 soon then